resources/localization/LCL/de/sql.xlf.lcl

```diff
@@ -12327,15 +12327,6 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
-      <Item ItemId=";src/sql/workbench/contrib/objectExplorer/common/serverGroup.contribution.serverTree.useAsyncServerTree" ItemType="0" PsrId="308" Leaf="true">
```

Comment:
FYI, you don't need to update any of the LCL/XLF files. That will be done automatically by Alex when he runs his localization scripts. So I'd just revert that to prevent any possible weirdness that manually updating them might cause.

Reply:
I was under the assumption that we must delete strings when we change the text with the same localization key.

Reply:
So technically if we want to be really on top of things you *should* change the loc key (essentially "deleting" the old key and adding a new one with a different name). But generally we don't bother doing this - as the normal localization process should ensure that the strings are kept all up to date. Which as a side note - you didn't do, since you just updated the text in the string but not the key itself. 

There is a small risk of having a string that was localized previously not get updated in time, thus for non-EN languages showing the wrong content. But we haven't gotten a complaint about that yet so it's not something we've bothered to do so far